pkg/discord/main.go

```diff
@@ -771,6 +795,145 @@ func fetchFile(data shared.CommandData) (*models.File, error) {
 	}
 }
 
+// Check common time formats
+func parseTime(timeStr string) (time.Time, error) {
+	now := time.Now()
+	formats := []string{
```

Comment:
Maybe force a format (ISO 8601) onto the user

Reply:
Depends on how performant you want it

Reply:
Yeah. I've debated a bit. Even if using the last date format that gets checked, the check only takes 10-20µs. So the debate is really simplicity of use vs. simplicity of code. What do you think?